pkgs/data/themes/everforest-gtk-theme/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  pname = "Everforest-GTK-Theme";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://github.com/Fausto-Korpsvart/Everforest-GTK-Theme/archive/refs/heads/master.tar.gz";
```

Comment:
This is not pinned, and the hash can change. Point to a specific commit instead (and use `fetchFromGitHub`).